### PR TITLE
Numbers man

### PR DESCRIPTION
**Pull request by stonedDiscord**
set numbers to be actual numbers and not strings with a single number in them

### CODE REVIEW

File: webAO/client.js [hunk 1]
```diff
@@ -1016,15 +1016,15 @@ class Client extends EventEmitter {
 		for (let i = 1; i <= ini.emotions.number; i++) {
 			const emoteinfo = ini.emotions[i].split("#");
 			const esfx = ini.soundn[i] ? ini.soundn[i] : "0";
-			const esfxd = ini.soundt[i] ? ini.soundt[i] : "0";
+			const esfxd = Number(ini.soundt[i] ? ini.soundt[i] : 0);
```
Comment (oldmud0) [inline webAO/client.js]:
```suggestion
			const esfxd = Number(ini.soundt[i] || 0);
```

File: webAO/client.js [hunk 2]
```diff
@@ -1016,15 +1016,15 @@ class Client extends EventEmitter {
 		for (let i = 1; i <= ini.emotions.number; i++) {
 			const emoteinfo = ini.emotions[i].split("#");
 			const esfx = ini.soundn[i] ? ini.soundn[i] : "0";
```
Comment (oldmud0) [inline webAO/client.js]: missed one?

Comment (stonedDiscord) [reply webAO/client.js]: soundN is the sfx name, 0 is used to indicate that there is no sfx to be played

Comment (oldmud0) [reply webAO/client.js]: le sigh

**Pull request merged**